cmd/client/sandbox.go

```diff
@@ -242,6 +268,32 @@ func ListPodSandboxes(client pb.RuntimeServiceClient) error {
 		fmt.Printf("Status: %s\n", pod.State)
 		ctm := time.Unix(*pod.CreatedAt, 0)
 		fmt.Printf("Created: %v\n", ctm)
+		if pod.Metadata != nil {
```

Comment:
nit.. status output sequence goes from id to metadata to status then created..
list output sequence goes from id to status to created then metadata.. 

Suggest picking one to use for both..